adafruit_pyportal.py

```diff
@@ -649,6 +654,33 @@ def image_converter_url(image_url, width, height, color_depth=16):
                                           width, height,
                                           color_depth, image_url)
 
+    def io_push(self, feed, data):
```

Comment:
How about `push_to_io` so it reads better?

Reply:
I prefer what I have, but if you want, I can change it

Reply:
Why do you like io_push? push_to_io seems grammatically correct to me.

Reply:
no reason, I've changed it